.github/workflows/Tests.yaml

```diff
@@ -20,6 +21,18 @@ on:
   push:
     branches-ignore:
       - gh-pages
+  # Once a month we run the tests and release a new version. The scheduled
+  # workflow always runs on the repository's default branch (`develop`).
+  schedule:
+    - cron: '0 0 1 * *'
+  # Also allow running the workflow manually. This can be used to manually
+  # release a version.
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: "Set to create a release on success (no 'v' prefix)"
```

Comment:
I'm not sure that I'm associating bits of this file correctly, but below I find
```
      # We use the current date as tag name, unless a tag name was specified
      # as input to the `workflow_dispatch` event
￼```
so if this is unset does it create a release with a default name or not create a release?

(I don't know why GitHub is refusing to format this comment.)

Reply:
It will create the release with the default name.